cinema/tests/test_movie_session_api.py

```diff
@@ -46,9 +46,7 @@ def test_get_movie_sessions(self):
         }
         self.assertEqual(movie_sessions.status_code, status.HTTP_200_OK)
         for field in movie_session:
-            self.assertEqual(
-                movie_sessions.data[0][field], movie_session[field]
-            )
+            self.assertEqual(movie_sessions.data[0][field], movie_session[field])
```

Comment:
Ensure that the URL '/api/cinema/movie_sessions/' matches the actual URL pattern defined in your Django project. Hardcoding URLs can lead to maintenance issues if the URL patterns change. Consider using Django's reverse function to dynamically generate URLs based on the view names.